README.md

```diff
@@ -1,2 +1,275 @@
+A JAVASCRIPT FILE (free to edit) - that handles functionalities and dynamic HTML
+
+### Prerequisites <a name="prerequisites"></a>
+> You should have Node and Git Installed
+> You should have basic knowledge on HTML / CSS / JavaScript
+> You should have a code editor
+
+### Install <a name="install"></a>
+
+> Download install [VSCODE](https://code.visualstudio.com/) and [Git](https://git-scm.com/)
+> To install linters, execute the following commands:
+```
+Initialize npm | ``` npminit -y ```
+HTML | ``` npm install --save-dev hint@7.x ```
+CSS | ```npm install --save-dev stylelint@13.x stylelint-scss@3.x stylelint-config-standard@21.x stylelint-csstree-validator@1.x  ```
+```
+### Usage <a name="usage"></a>
+To run the linters, execute the following command and fix linter errors:
+```
+HTML | ``` npx hint . ```
+CSS | ```npx stylelint "**/*.{css,scss}" ```
+If you get a flood of errors keep in mind that linters guide you in writing a clean code!
+```
+### Run Tests <a name="run-tests"></a>
+You can run this program on your browser
+
+### Deployment <a name="deployment"></a>
+
+You can deploy this project using the following procedure:
+
+- Pick the right hosting provider.
+- Choose the tool and method to upload your website.
+- Upload files to your website.
+- Move the website files to the main root folder.
+- Import your database.
+- Check if your website works worldwide.
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- AUTHORS -->
+
+## 👥 Authors <a name="authors"></a>
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+👤 **Eloghene Otiede**
+
+- GitHub: [@geekelo](https://github.com/geekelo)
+- Twitter: [@Geekelo_xyz](https://twitter.com/Geekelo_xyz)
+- LinkedIn: [LinkedIn](https://linkedin.com/in/eloghene-otiede)
+[See Documentation of Author's inputs](https://geekelo.com.ng/crypto-price-rank-project-tutorials/)
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- Contributor -->
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+## 🤝 Contributing <a name="contributing"></a>
+
+You can offer Contributions, submit an [issue](../../issues/), and make a feature request.
+
+## Future Features <a name="future-features"></a>
+
+- Add pop up modals
+- Add the contact form 
+- Add more content pages
+
+## ⭐️ Show your support <a name="support"></a>
+
+> Please give a ⭐️ to support this project
+
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- ACKNOWLEDGEMENTS -->
+
+## 🙏 Acknowledgments <a name="acknowledgements"></a>
+
+- > I would like to thank Microverse for inspiring this project 
+- > Fonts: [Google Fonts](https://fonts.google.com/)
+- > Original design idea by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio).
+
+<p align="right">(<a href="#readme-top">back to top</a>)</p>
+
+<!-- LICENSE -->
+
+## 📝 License <a name="license"></a>
+
+This project is [MIT](./LICENSE) licensed.
+
```

Comment:
- [ ] The link to your license in your readme file is no longer active. Please correct it. It is critical to include a license in your project because it is a document that provides legally binding guidelines for use and imposes restrictions on how your project can be used. 📗 🙂

![Screenshot from 2023-08-19 03-13-06](https://github.com/geekelo/crypto-price-rank/assets/89228743/9094c409-8456-438d-bbaf-b3b7da619536)